scenarioo-server/src/main/java/org/scenarioo/rest/step/ScreenshotResource.java

```diff
@@ -66,19 +66,39 @@ public ResponseEntity getScreenshot(@PathVariable("branchName") final String bra
 	 * This method is used for sharing screenshot images. It is a bit slower, because the image filename has to be
 	 * resolved first. But it is also more stable, because it uses the new "stable" URL pattern.
 	 */
-	@GetMapping(path = "{scenarioName}/pageName/{pageName}/pageOccurrence/{pageOccurrence}/stepInPageOccurrence/{stepInPageOccurrence}/image.{extension}", produces = "image/jpeg")
-	public ResponseEntity getScreenshotStable(@PathVariable("branchName") final String branchName,
+	@GetMapping(path = "{scenarioName}/pageName/{pageName}/pageOccurrence/{pageOccurrence}/stepInPageOccurrence/{stepInPageOccurrence}/image.png", produces = "image/png")
+	public ResponseEntity getScreenshotStablePng(@PathVariable("branchName") final String branchName,
 											  @PathVariable("buildName") final String buildName, @PathVariable("usecaseName") final String usecaseName,
 											  @PathVariable("scenarioName") final String scenarioName, @PathVariable("pageName") final String pageName,
 											  @PathVariable("pageOccurrence") final int pageOccurrence,
 											  @PathVariable("stepInPageOccurrence") final int stepInPageOccurrence,
 											  @RequestParam(value="fallback", required = false) final boolean fallback, @RequestParam(value="labels", required = false) final String labels) {
 
+		return getScreenshotStable(branchName, buildName, usecaseName, scenarioName, pageName, pageOccurrence, stepInPageOccurrence, fallback, labels);
+	}
+
+	/**
+	 * This method is used for sharing screenshot images. It is a bit slower, because the image filename has to be
+	 * resolved first. But it is also more stable, because it uses the new "stable" URL pattern.
+	 * Additional method with produces=image/jpg is needed, as firefox is a bit picky in its accept headers, so produces image/* does not work.
+	 */
+	@GetMapping(path = "{scenarioName}/pageName/{pageName}/pageOccurrence/{pageOccurrence}/stepInPageOccurrence/{stepInPageOccurrence}/image.{extension}", produces = "image/jpg")
```

Comment:
before the content type was "image/jpeg" - and i think this would be more correct - according to https://stackoverflow.com/questions/33692835/is-the-mime-type-image-jpg-the-same-as-image-jpeg

@sascha-kaufmann @eggerschwiler